ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -12,18 +12,21 @@ import {services} from '../../../shared/services';
 import {WorkflowListItem} from '..';
 import {BasePage} from '../../../shared/components/base-page';
 import {Loading} from '../../../shared/components/loading';
-import {NamespaceFilter} from '../../../shared/components/namespace-filter';
 import {Query} from '../../../shared/components/query';
 import {ResourceSubmit} from '../../../shared/components/resource-submit';
 import {ZeroState} from '../../../shared/components/zero-state';
 import {exampleWorkflow} from '../../../shared/examples';
 import {Utils} from '../../../shared/utils';
 
+import {WorkflowFilters} from '../workflow-filters/workflow-filters';
+
 require('./workflows-list.scss');
 
 interface State {
     loading: boolean;
     namespace: string;
+    phases: string[];
```

Comment:
Doesn't phase have a TS type? See: https://github.com/argoproj/argo/blob/8811231299434e89ee9279e400db3445d83fec39/ui/src/models/workflows.ts#L903

This should be
```suggestion
    phases: NodePhase[];
```